app/controllers/users.ts

```diff
@@ -8,10 +8,16 @@ import { notFoundError, databaseError, authenticationError } from '../errors';
 import { encode } from '../services/session';
 
 export function getUsers(req: Request, res: Response, next: NextFunction): Promise<Response | void> {
+  const page: number = Number(req.query.page) || 1;
```

Comment:
What would happen if i send a negative value?

Reply:
i added a condition for this case